src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Security/AuthorizationHandlers/ViewDataSetVersionAuthorizationHandlerTests.cs

```diff
@@ -90,15 +90,20 @@ public async Task Success_PreviewTokenIsActive(DataSetVersionStatus status)
         Assert.True(context.HasSucceeded);
     }
 
-    [Fact]
-    public async Task Failure_PreviewTokenIsExpired()
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public async Task Failure_PreviewTokenIsExpiredOrNotActiveYet(bool toggleBoolean)
     {
         DataSetVersion dataSetVersion = _dataFixture
             .DefaultDataSetVersion()
             .WithStatus(DataSetVersionStatus.Draft);
 
         PreviewToken previewToken = _dataFixture
-            .DefaultPreviewToken(expired: true)
+            // Test when activated is true and expired is also true.
+            // Also test when activated is not yet & expired is also not yet.
+            // Both result in non-valid tokens 
+            .DefaultPreviewToken(activated: toggleBoolean, expired: toggleBoolean)
```

Comment:
Having one `toggleBoolean` parameter controlling both doesn't check a third case when activated is false and expired is true.

I think you could have two parameters like `isActive` and `isExpired` with data

`false` / `false`
`fasle` / `true`
`true` / `true`